src/app/dim-ui/EnergyMeterIncrements.scss

```diff
@@ -0,0 +1,89 @@
+// <div class=`energyMeterIncrements small`> (or medium)
+//   <div class=used/><div class=used/><div class=unused/><div class=unavailable/><div class=unavailable/>
+// </div>
+//
+// results:
+// ■ ■ ■ □ ▬ ▬
+
+.energyMeterIncrements {
+  display: flex;
+  align-items: center;
+
+  .unavailable,
+  .used,
+  .unused {
+    display: block;
+    flex-grow: 1;
+    border-style: solid;
+  }
+  .unavailable {
+    height: 0;
+  }
+  &.small {
+    .unavailable,
+    .used,
+    .unused {
+      border-width: 1px;
+      margin: 0 0.5px;
+    }
+
+    .used,
+    .unused {
+      height: 2px;
+    }
+  }
+
+  &.medium {
+    .unavailable,
+    .used,
+    .unused {
+      border-width: 3px;
+      margin: 0 1px;
+    }
+
+    .used,
+    .unused {
+      height: 6px;
+    }
+  }
+
+  .unavailable {
+    border-color: #888;
+  }
+
+  &.solar > .used {
+    border-color: #f0631e;
+    background: #f0631e;
+  }
+
+  &.solar > .unused {
+    border-color: #f0631e;
+  }
+
+  &.arc > .used {
+    border-color: #79bbe8;
```

Comment:
I noticed that these differ from the arc, void, solar colours in our css variables. Should we unify them or not?

Reply:
probably yeah, but some versions of the colors are slightly modified for readability in their particular area, so it needs love and care